ndarbas10/startMokinys2.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require 'Mokinys.php';
+$mokiniai = [
+    new Mokinys(['lietuviu' => 8, 'matematika' => 9, 'anglu' => 8], '3a', 'Jonas', 'Jonaitis'),
+    new Mokinys(['lietuviu' => 9, 'matematika' => 10, 'anglu' => 10], '2b', 'Ona', 'Onaitė'),
+    new Mokinys(['lietuviu' => 8, 'matematika' => 7, 'anglu' => 9], '2c', 'Antanas', 'Antanaitis'),
+    new Mokinys(['lietuviu' => 9, 'matematika' => 9, 'anglu' => 10], '2b', 'Eglė', 'Eglytė'),
+];
+
+echo '<h4>Pažymių vidurkiai, mžėjimo tvarka</h4>';
+
+$rezult = [];
+$i=0;
+foreach ($mokiniai as $mokinys) {
+    $rezult[$i] = [round($mokinys->vidurkis(),2),$mokinys->vardas, $mokinys->pavarde];
```

Comment:
Vietoj `2` galejote naudoti `PHP_ROUND_HALF_DOWN` konstanta, tam ji ir yra.

Reply:
Dabar žinau, kad tokia yra.

Reply:
Konstanta PHP_ROUND_HALF_DOWN nusako apvalinimo būdą, skaičius '2' - apvalinimo tikslumą. Tai neapkeičiami round parametrai. Be to PHP_ROUND_HALF_DOWN yra parametras pagal nutylėjimą ir naudoti nėra prasmės.

Reply:
Supratau, sumaisiau antra parametra su treciu. :+1: